janusgraph-es/src/test/resources/equalsOrContainsCustomMapping.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
@davidclement90 All the updates look good except this file is still there. Is it necessary or can it be removed?